engine/src/main/java/org/pentaho/di/trans/steps/pentahoreporting/PentahoReportingOutputMeta.java

```diff
@@ -138,6 +146,10 @@ private void readData( Node stepnode ) throws KettleXMLException {
     try {
       inputFileField = XMLHandler.getTagValue( stepnode, "input_file_field" );
       outputFileField = XMLHandler.getTagValue( stepnode, "output_file_field" );
+      inputFile = XMLHandler.getTagValue( stepnode, "input_file" );
```

Comment:
Define a constant instead of duplicating all these literals X times

Reply:
done